examples/DynaSwing/GridForming_GridFollowing/fic.JOB

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<jobs xmlns="http://www.rte-france.com/dynawo">
+    <job name="All converter-interfaced 5-machine system (3 GFM + 2 GFL)">
+        <solver lib="dynawo_SolverSIM" parFile="fic.PAR" parId="SimplifiedSolver"/>
```

Comment:
Is there a specific reason for using the simplified solver instead of IDA here?

Reply:
Yes, with IDA, I cannot pass the fault (sqrt < 0) unless RFault is extremely high